sw/device/lib/testing/flash_ctrl_testutils.h

```diff
@@ -25,7 +25,23 @@ bool flash_ctrl_testutils_wait_transaction_end(
     dif_flash_ctrl_state_t *flash_state);
 
 /**
- * Setup and enable for a data region.
+ * Setup and enable for a data region taking region properties as a parameter.
+ * Returns the address offset of the region.
```

Comment:
I would drop this line, as there`s a return description below.

Reply:
Thanks @davebwil for the changes, I know it was there already, but could just drop this line here as well for consistency?

Reply:
Thanks @engdoreis. Changed this one as well.